index.html

```diff
@@ -5,10 +5,23 @@
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
   <meta http-equiv="X-UA-Compatible" content="ie=edge">
   <title>Contact Manager</title>
+  <script src="https://kit.fontawesome.com/634f0d54fa.js" crossorigin="anonymous"></script>
+ <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css">
   <link rel="stylesheet" href="./assets/style/main.css">
 </head>
 <body>
-  
+  <div class="header row">
+    <div class="app-title ">
+      <h2>Contact Manager</h2>
```

Comment:
Use <h1> tag for SEO purposes.

Reply:
@ATOGUNEFE 
I meant that the `<h1>` tag should be used instead of `<h2>`. This is important for Search engine optimization. The `<h1>` tag would be identified as a major title in the page - which it is.